src/_provisional.scss

```diff
@@ -149,6 +149,528 @@
+
+	.btn {
+		@extend %gcweb-btn;
+	}
+
+	.btn-lg {
+		@extend %gcweb-btn-lg;
+	}
+
+	.btn-sm {
+		@extend %gcweb-btn-sm;
+	}
+
+	.btn-xs {
+		@extend %gcweb-btn-xs;
+	}
+
+	/*keep post-finalization*/
+	.btn-call-to-action {
+		@extend %gcweb-btn-call-to-action-margin;
+	}
+
+
+
+	/*Dropdown menus... (derived from https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_dropdowns.scss)*/
+
+	.caret {
+		border-left: $gcweb-caret-width-base solid transparent;
+		border-right: $gcweb-caret-width-base solid transparent;
+		border-top: $gcweb-caret-width-base dashed;
+	}
+
+	.dropup,
+	.navbar-fixed-bottom .dropdown {
+		.caret {
+			border-bottom: $gcweb-caret-width-base dashed;
+			border-top: 0;
+		}
+	}
+
+
+
+	/*Button groups... (derived from https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_button-groups.scss)*/
+
+	.btn-group > .btn:not(:first-child):not(:last-child):not(.dropdown-toggle) {
+		border-radius: 0;
+	}
+
+	.btn-group > .btn:first-child {
+		margin-left: 0;
+		&:not(:last-child):not(.dropdown-toggle) {
+			@include border-right-radius(0);
+		}
+	}
+
+	.btn-group > .btn:last-child:not(:first-child),
+	.btn-group > .dropdown-toggle:not(:first-child) {
+		@include border-left-radius(0);
+	}
+
+	.btn-group > .btn-group:not(:first-child):not(:last-child) > .btn {
+		border-radius: 0;
+	}
+
+	.btn-group > .btn-group:first-child:not(:last-child) {
+		> .btn:last-child,
+		> .dropdown-toggle {
+			@include border-right-radius(0);
+		}
+	}
+
+	.btn-group > .btn-group:last-child:not(:first-child) > .btn:first-child {
+		@include border-left-radius(0);
+	}
+
+	.btn-lg .caret {
+		border-bottom-width: 0;
+		border-width: $gcweb-caret-width-large $gcweb-caret-width-large 0;
+	}
+
+	.dropup .btn-lg .caret {
+		border-width: 0 $gcweb-caret-width-large $gcweb-caret-width-large;
+	}
+
+	.btn-group-vertical > .btn {
+		&:not(:first-child):not(:last-child) {
+			border-radius: 0;
+		}
+
+		&:first-child:not(:last-child) {
+			@include border-top-radius($btn-border-radius-base);
+			@include border-bottom-radius(0);
+		}
+
+		&:last-child:not(:first-child) {
+			@include border-top-radius(0);
+			@include border-bottom-radius($btn-border-radius-base);
+		}
+	}
+
+	.btn-group-vertical > .btn-group:not(:first-child):not(:last-child) > .btn {
+		border-radius: 0;
+	}
+
+	.btn-group-vertical > .btn-group:first-child:not(:last-child) {
+		> .btn:last-child,
+		> .dropdown-toggle {
+			@include border-bottom-radius(0);
+		}
+	}
+
+	.btn-group-vertical > .btn-group:last-child:not(:first-child) > .btn:first-child {
+		@include border-top-radius(0);
+	}
+
+
+
+	/*Forms... (derived from https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_forms.scss)*/
+
+	/*keep post-finalization*/
+	.checkbox,
+	.checkbox-inline,
+	.radio,
+	.radio-inline {
+		@extend %gcweb-main-font-size-line-height;
+	}
+
+	legend {
+		font-size: ($gcweb-font-size-base * 1.5);
```

Comment:
My rationale is that's what Bootstrap does lol. Legends only look like regular text when the ``chkbxrdio-grp`` class is used on their parent ``<fieldset>``.

This is a copy of Bootstrap's ``legend`` selector that uses a GCWeb-specific variable instead of Bootstrap's equivalent one:
https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_forms.scss#L25

FWIW that's how most of this PR works. I basically copied all of Bootstrap's form selectors, gutted most of their properties and only kept ones that needed to be re-declared as overrides (primarily those that use variables). Plus some extra selectors to deal with quirks in WET/GCWeb.

As for the design of legends in general, IMO it is strange how Bootstrap decided to make its legends look so big :S. But it should be kept in mind that Bootstrap's base font size is 14px. In that context the legend's font size (21px) is bigger than normal text, but still looks clearly smaller than headings. By comparison, WET increased the base font size to 16px while keeping heading sizes as-is. As a result, WET's legends look almost like high-level headings. GCWeb's legends currently look the same as in WET. But with this PR, they'd balloon to 30px since GCWeb's base font is 20px.